sos/report/plugins/openstack_octavia.py

```diff
@@ -60,7 +66,16 @@ def setup(self):
         for loadbalancer in loadbalancers.splitlines():
             loadbalancer = loadbalancer.split()[0]
             self.add_cmd_output(
-                "openstack loadbalancer show %s" % (loadbalancer)
```

Comment:
This is a big change. The status tree is useful for sure, but by not capturing the load balancer configuration (flavor, az, etc.) I think we are losing a lot of useful information. I'm on this fence on this one.

Reply:
Good point. My first thought is always that "status show" dumps everything which is not true. Do you think it is best to just do a "loadbalancer show" or both show and status?

Reply:
I reverted this change back to "loadbalancer show". This output combined with other resource show commands should provide all needed information.